src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,374 @@
+@import "../../patternfly-utilities";
+
+.pf-c-sequence {
+  --pf-c-sequence--BackgroundColor: var(--pf-global--BackgroundColor--dark-100); // this background color is interchangable
+
+  // Sequence Control
+  --pf-c-sequence__control--PaddingTop: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--PaddingBottom: var(--pf-global--spacer--xl);
+  --pf-c-sequence__control--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-sequence__control--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--BorderTopColor: transparent;
+  --pf-c-sequence__control--BorderBottomColor: transparent;
+  --pf-c-sequence__control--hover--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--active--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--focus--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--hover--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--active--BorderTopHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--focus--BorderBottomHeight: var(--pf-global--BorderWidth--lg);
+  --pf-c-sequence__control--hover--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--active--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--focus--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--hover--BorderBottomColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--active--BorderTopColor: var(--pf-global--primary-color--100);
+  --pf-c-sequence__control--focus--BorderBottomColor: var(--pf-global--primary-color--100);
+
+  // Sequence Control Left
+  --pf-c-sequence__control-left--PositionLeft: 0;
+  --pf-c-sequence__control-left--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-sequence__control-left--BorderRightWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-sequence__control-left--BorderRightColor: var(--pf-global--Color--200);
+
+  // Sequence Control Right
+  --pf-c-sequence__control-right--PositionRight: 0;
+  --pf-c-sequence__control-right--BackgroundColor: var(--pf-global--BackgroundColor--dark-100);
+  --pf-c-sequence__control-right--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-sequence__control-right--BorderLeftColor: var(--pf-global--Color--200);
+
+  // Sequence Control Icon
+  --pf-c-sequence__control-icon--Color: var(--pf-global--Color--light-100);
+
+  // Item
+  --pf-c-sequence__item--MarginRight: 1px;
+  --pf-c-sequence__item--MarginLeft: 1px;
+  --pf-c-sequence__item--MaxWidth: 200px;
```

Comment:
Can you change this to `#{pf-size-prem(200px)};`?

Reply:
yes, changed this